src/html2md.js

```diff
@@ -84,13 +89,54 @@ function assertValidJSON(str) {
   }
 }
 
+/**
+ * @param {string} str
+ * @param {number} [limit]
+ * @returns {string}
+ * @throws {Error}
+ */
 function assertMetaSizeLimit(str, limit = 128_000) {
   if (str && str.length > limit) {
     throw Error('metadata size limit exceeded');
   }
   return str;
 }
 
+/**
+ * Check if meta name is allowed:
+ *  - non-reserved
+ *  - not starting with 'twitter:'
+ *    - except 'twitter:label' and 'twitter:data'
```

Comment:
but the logic in html-pipeline is different:

```
    const attr = name.includes(':') && !name.startsWith('twitter:') ? 'property' : 'name';
```

if use `name` if:
- name does not include a `:`
- or if starts with `twitter:`

Reply:
I think that is consistent with this change.. the code previously disallowed any `twitter:` properties to be provided, this just allows some through (specifically `twitter:label*` and `twitter:data*`. Those are used for Slack link unfurling, for example:

```
<meta name="twitter:label1" content="Price">
<meta name="twitter:data1" content="799">
<meta name="twitter:label2" content="Availability">
<meta name="twitter:data2" content="In stock">
```

results in:
<img width="669" alt="image" src="https://github.com/user-attachments/assets/b6fc1db1-c2e1-42dd-a00d-9d1e17ef08a6">